src/config/dynamic-theme-fixes.config

```diff
@@ -4668,6 +4675,15 @@ CSS
 
 ================================
 
+libretexts.org
+
+CSS
+.mt-content-container img {
```

Comment:
What about 
```CSS
INVERT
.internal[alt$=".svg"]
```
It should only target the `math` graphs.

Reply:
I thought it best to target all images in the text body, as there might be images of other formats with transparent backgrounds. The writing [here](https://chem.libretexts.org/Bookshelves/Ancillary_Materials/Reference/Organic_Chemistry_Glossary/Resonance_Energy) has a PNG image with a transparent background above the line "The difference in energy between X and X3 is the resonance energy of X."
And I used the `.mt-content-container` selector to target only the images in the main text body; I was being conservative and didn't want to break anything I wasn't aware of. Is that unnecessary?

Also, the reason I used CSS instead of `INVERT` is that color-coded images may be adversely affected. LibreTexts hosting academic content, I thought that isn't unlikely.

Reply:
`I was being conservative and didn't want to break anything I wasn't aware of. Is that unnecessary?`

The preference goes to INVERT instead of giving users a white background-color.
We encoutered this problem with Wikipedia trough:
```CSS
.mwe-popups-discreet > svg,
div .thumbimage[src$=".png"],
div .thumbimage img[src$=".png"] {
    background-color: white;
}
.mw-mmv-image .svg,
.fullImageLink [src*=".svg"],
a[href$=".svg"]:hover > img,
a[href*=".gif"]:hover > img {
    background-color: rgba(255, 255, 255, 0.75) !important;
    background-blend-mode: color;
}
```
Something similair can be implemented for libretexts to ensure it will not need a white background-color.

Reply:
Do you mean using a light grey background instead of a white background?

Reply:
> Do you mean using a light grey background instead of a white background?

Only while hovering due to some issues we encountered that still can happen on white background.

Reply:
So just to confirm, what you're suggesting is to leave the images unaltered, and to give them a light background only when they are hovered over. Is that correct?

Reply:
That's correct.